app.py

```
@@ -0,0 +1,6870 @@
+        if "user_type" in request.GET and len(request.GET["user_type"]) > 0:
+
+            if (
+                int(request.GET["user_type"]) == 3
+                and request.GET["applicant_only"] == "1"
+            ):
+
+                data = data.filter(can_source_id=request.GET["user_type"]).exclude(
+                    applicant__isnull=True
+                )
+            elif (
+                int(request.GET["user_type"]) == 3
+                and request.GET["applicant_only"] == "0"
+            ):
+                data = data.filter(can_source_id=request.GET["user_type"])
+            else:
+                data = data.filter(can_source_id=request.GET["user_type"])
+            user_type = request.GET["user_type"]
+        else:
+            user_type = ""
+        if "skill_match" in request.GET and len(request.GET["skill_match"]) > 0:
+            skill_match_list = request.GET["skill_match"].split(",")
+            data = data.filter(
+                reduce(
+                    operator.or_,
+                    (Q(skills__icontains=item) for item in skill_match_list),
+                )
+            )
+        if "sort" in request.GET and len(request.GET["sort"]) > 0:
+            if request.GET["sort"] == "interested":
+                data = data.order_by("-interested", "-match")
+            elif request.GET["sort"] == "not_interested":
+                data = data.order_by("-responded_date", "interested", "-match")
+            elif request.GET["sort"] == "name":
+                data = data.order_by("first_name")
+            elif request.GET["sort"] == "invited":
+                data = data.order_by("-invite")
+            elif request.GET["sort"] == "not_invited":
+                data = data.order_by("invite")
+
+        search = False
+        total_count = data.count()
+        try:
+            if request.GET["search"] == "1":
+                search = True
+        except:
```

Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and addressed. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the [logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception) API. For example,

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```